client/client.go

```diff
@@ -1695,6 +1697,32 @@ func (c *client) scatterRegionsWithOptions(ctx context.Context, regionsID []uint
 	return resp, nil
 }
 
+func (c *client) SplitAndScatterRegions(ctx context.Context, splitKeys [][]byte, opts ...RegionsOption) (*pdpb.SplitAndScatterRegionsResponse, error) {
+	start := time.Now()
+	defer func() { cmdDurationScatterRegions.Observe(time.Since(start).Seconds()) }()
```

Comment:
Is it suitable to use `cmdDurationScatterRegions` for `SplitAndScatterRegions`? Maybe you need to add `cmdDurationSplitAndScatterRegions` for it.